functions.opengraph.php

```diff
@@ -245,7 +245,12 @@ function jetpack_og_get_image( $width = 200, $height = 200, $max_images = 4 ) {
 		$image['src'] = jetpack_site_icon_url( null, '512' );
 	}
 
-	// Fourth fall back, blank image
+	// Fourth fall back, Core Site Icon. Added in WP 4.3.
+	if ( empty( $image ) && ( function_exists( 'has_site_icon') && has_site_icon() ) ) {
```

Comment:
I think this could be done a bit better since jetpack_site_icon_url and get_site_icon_url should always return the same thing. 

We can do a check if one exist and it it doesn't call the `jetpack_site_icon_url` function.